doc/dev/deduplication.rst

```diff
@@ -157,25 +157,46 @@ How to use deduplication
 Ceph provides deduplication using RADOS machinery.
 Below we explain how to perform deduplication. 
 
+Prerequisite
+------------
+
+If the Ceph cluster is started from Ceph mainline, users need to check
+``ceph-test`` package which is including ceph-dedup-tool is installed.
+
+Deatiled Instructions
+---------------------
+
+Users can use ceph-dedup-tool with ``estimate``, ``sample-dedup``, 
+``chunk-scrub``, and ``chunk-repair`` operations. To provide users with
+high flexibility, we have enabled necessary operations through
+ceph-dedup-tool, and we recommend using the following operations freely
+by using any types of scripts.
+
 
 1. Estimate space saving ratio of a target pool using ``ceph-dedup-tool``.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. code:: bash
 
-    ceph-dedup-tool --op estimate --pool $POOL --chunk-size chunk_size  
-      --chunk-algorithm fixed|fastcdc --fingerprint-algorithm sha1|sha256|sha512
-      --max-thread THREAD_COUNT
+    ceph-dedup-tool --op estimate
+      --pool [BASE_POOL]
+      --chunk-size [CHUNK_SIZE]
+      --chunk-algorithm [fixed|fastcdc]
+      --fingerprint-algorithm [sha1|sha256|sha512]
+      --max-thread [THREAD_COUNT]
 
 This CLI command will show how much storage space can be saved when deduplication
 is applied on the pool. If the amount of the saved space is higher than user's expectation,
 the pool probably is worth performing deduplication. 
-Users should specify $POOL where the object---the users want to perform
+Users should specify ``BASE_POOL`` where the object---the users want to perform
```

Comment:
@zdover23: I think that this parenthetical statement set apart by dashes intends to say "the object upon which users want to perform deduplication". If so, it will not escape the attention of the careful reader that such a clause is in want of more elegant wording. Perhaps something like "Users should specify the ``BASE_POOL``, within which the object targeted for deduplication is stored." If pragmatically correct, this recast sentence is an improvement though it may not be euphonious.

Reply:
Thank you for suggestion.
"Users should specify the BASE_POOL, within which the object targeted for deduplication is stored."
It looks good for me. I'll fix it.